app/code/core/Mage/Media/Model/Image.php

```diff
@@ -72,7 +74,7 @@ protected function _construct()
      * Set media image config instance
      *
      * @param Mage_Media_Model_Image_Config_Interface $config
-     * @return unknown
+     * @return Mage_Media_Model_Image
```

Comment:
Shouldn't it be `@return $this`?

Reply:
Thsi was right, but collin merged some seconds ago ... will be fixed in "Round 2".

Reply:
Thanks for your effort.